internal/test/go.sum

```diff
@@ -1,81 +1,57 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
 cloud.google.com/go v0.38.0/go.mod h1:990N+gfupTy94rShfmMCWGDn0LpTmnzTp2qbd1dvSRU=
-github.com/Azure/azure-sdk-for-go v34.1.0+incompatible h1:uW/dgSzmRQEPXwaRUN8WzBHJy5J2cp8cw1ea908uFj0=
```

Comment:
A bit confused by this. Why are we seeing a change in `go.sum`?

Reply:
We're seeing a change in go.sum as the k8s and openshift libraries have been updated, changing the subdependencies they're bringing in. That change is reflected here in the go.sum

Reply:
So why isn't this going into the previous bump commit?